snippets/standard/buffers/memory-t/owner/owner.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Buffers;
+
+class Example
+{
+    static void Main()
+    {
+        IMemoryOwner<char> owner = MemoryPool<char>.Shared.Rent();
+
+        Console.Write("Enter a number: ");
+	    try {
+            var value = Int32.Parse(Console.ReadLine());
+
+            var memory = owner.Memory;
+
+            WriteInt32ToBuffer(value, memory);
+
+            DisplayBufferToConsole(owner.Memory.Slice(0, value.ToString().Length));
+        }
+        catch (FormatException) {
+            Console.WriteLine("You did not enter a valid number.");
+        }
+        catch (OverflowException) {
+            Console.WriteLine($"You entered a number less than {Int32.MinValue:N0} or greater than {Int32.MaxValue:N0}.");
+        }
+        finally {
+            owner.Dispose();
```

Comment:
minor nit:  I'd prefer `owner?.Dispose();` to protect against the statement in line 8 throwing.